frame/nomination-pools/benchmarking/src/lib.rs

```diff
@@ -262,16 +283,17 @@ frame_benchmarking::benchmarks! {
 
 	}: bond_extra(RuntimeOrigin::Signed(scenario.creator1.clone()), BondExtra::Rewards)
 	verify {
+		 // commission of 50% deducted here.
 		assert!(
 			T::Staking::active_stake(&scenario.origin1).unwrap() >=
-			scenario.dest_weight
+			scenario.dest_weight / 2u32.into()
 		);
 	}
 
 	claim_payout {
 		let origin_weight = Pools::<T>::depositor_min_bond() * 2u32.into();
 		let ed = CurrencyOf::<T>::minimum_balance();
-		let (depositor, pool_account) = create_pool_account::<T>(0, origin_weight);
+		let (depositor, pool_account) = create_pool_account::<T>(0, origin_weight, Some(Perbill::from_percent(50)));
```

Comment:
Added 50% commission here, but the verify stage still passes with the depositor having the same free balance.

Reply:
Nvm, the depositor is also the payee so that account will still get the full funds.